### ISSUE TITLE
Using a proxy server when getting options data

### ISSUE THREAD

**Issue by uk-medic-coder** [ranaroussi/yfinance#2077]
Hi, is it possible to specify a proxy server with "yf.Ticker('AAPL')"?

I am downloading options data and would like to use a proxy.

Thank you.

**Comment by nickdon2007** [comment]
Yeah, I want to know if this is a feasible option. Download option data is too bulk of data and it is better to do with proxy.

**Comment by ValueRaider** [comment]
https://ranaroussi.github.io/yfinance/advanced/config.html
